11-if/02-if-statement/05-challenge-userpass/02-2nd-challenge/01-challenge/main.go

```diff
@@ -51,6 +43,8 @@ const (
 	accessOK = "Access granted to %q.\n"
 	user     = "jack"
```

Comment:
name it consistently like user1 and user2 rather than user and user2

Reply:
Fixed for user and password